sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/QueryPlanningTracker.scala

```diff
@@ -134,7 +134,7 @@ class QueryPlanningTracker(
    * and the recorded end time will be the end time of the last call.
    */
   def measurePhase[T](phase: String)(f: => T): T = {
-    val startTime = System.currentTimeMillis()
+    val startTime = System.currentTimeMillis
```

Comment:
Is this a known compilation error that will be triggered in Scala 3? Or is it an existing compilation warning in Scala 2.13? If not, I don't think it's worth fixing it just for the sake of maintaining consistency in call style.

Reply:
no error or warning，just a code refactor for it，starttime with() and endtime not，is a little strange @LuciferYang

Reply:
then let's just leave it.

Reply:
ok, close it